filter.imp

```diff
@@ -96,21 +96,11 @@
 
 
    # The below headers were added by hand due to errors being found when the private headers were included.
-   { include: ["@.asm/bitsperlong.h.", private, "<linux/bits.h>", "public"] },
-   { include: ["@.asm/errno.h.", private, "<linux/errno.h>", "public"] },
-   { include: ["@.asm/resource.h.", private, "<linux/resource.h>", "public"] },
+   # Additionally, If an asm header has no asm-generic version and is not present in every architecture 
+   # like pgtable_64_types we can use a non-architecture specific header instead
```

Comment:
I think now the location of this comment in this file is confusing, as the very first rule after it is an asm-generic.

Should the below rules that are asm-generic be moved ABOVE this comment?  Then the asm ones remain below the comment, then I think the comment might make more sense?